### PR TITLE
delete Temp files on clear button pressed

### PR DESCRIPTION
**Pull request by milosilic**
When a file is uploaded using FileUpload2 or FileUploadMultiple and then cleared from the element using clear button, it will remain hanging in temp folder of server. 
This PR should fix that

### CODE REVIEW

File: to.etc.domui/src/main/java/to/etc/domui/component/upload/FileUpload2.java [hunk 1]
```diff
@@ -313,6 +314,7 @@ private void clearClicked() throws Exception {
 	public void clear() throws Exception {
 		if(m_value == null)
 			return;
+		FileTool.closeAll(m_value.getFile());
```
Comment (millevlada) [inline to.etc.domui/src/main/java/to/etc/domui/component/upload/FileUpload2.java:317]: since it might be unclear why is that needed, we could add an comment next to it:
//removes created temp file

File: to.etc.domui/src/main/java/to/etc/domui/component/upload/FileUploadMultiple.java [hunk 2]
```diff
@@ -191,6 +192,7 @@ private void renderValue(Div valueD, UploadItem uploadItem) {
 		d.add(uploadItem.getRemoteFileName());
 		btn.setClicked(a -> {
 			m_value.remove(uploadItem);
+			FileTool.closeAll(uploadItem);
```
Comment (millevlada) [inline to.etc.domui/src/main/java/to/etc/domui/component/upload/FileUploadMultiple.java:195]: same here, comment would help to understand why this code is useful

File: to.etc.domui/src/main/java/to/etc/domui/component/upload/FileUploadMultiple.java [hunk 3]
```diff
@@ -245,10 +247,10 @@ private void validate() {
 		}
 	}
 
-	@Override public void setValue(@Nullable List<UploadItem> value) {
+	@Override
```
Comment (millevlada) [inline to.etc.domui/src/main/java/to/etc/domui/component/upload/FileUploadMultiple.java:250]: this was not necessary change, we try not to change unchanged code

**Review by fjalvingh — APPROVED** [review]
